src/Cli/dotnet/Telemetry/TelemetryCommonProperties.cs

```diff
@@ -54,6 +57,7 @@ internal class TelemetryCommonProperties
 
         private const string MachineIdCacheKey = "MachineId";
         private const string IsDockerContainerCacheKey = "IsDockerContainer";
+        private const string CI = "Continuous Integration";
```

Comment:
Should the space be removed? It seems to serve as a key and it appears we don't use spaces for these  (looking at MachineId and IsDockerContainer)

Reply:
I had the same question but some of the other keys do have spaces.

Reply:
This is a good suggestion, I'll do so.

Reply:
_Actually_, the telemetry property keys all use spaces. The keys without spaces are local cache keys only. I'll move this key up with the other telemetry property keys to reduce visual confusion.